### ISSUE TITLE
[css‑text] Rename `text‑wrap: nowrap` to `text‑wrap: no‑wrap`

### ISSUE THREAD

**Issue by ExE-Boss** [w3c/csswg-drafts#5376]
See the first point of the [List of Legacy Mistakes in the Design of CSS](https://wiki.csswg.org/ideas/mistakes).

I don’t think we should repeat that mistake again.

**Comment by Loirooriol** [comment]
Too many sites rely on `nowrap`, it can't be changed. Adding `no-wrap` as an alias would be possible, but was already rejected in #1537. Do you have new arguments to reconsider that resolution?

**Comment by ExE-Boss** [comment]
This isn’t about `white‑space`, this is about the new **CSS Text 4** `text‑wrap` property, which no engines are shipping yet.

**Comment by frivoal** [comment]
`text-wrap` is a longhand of `white-space` though, so `nowrap` is the same keyword with the same meaning. Shorthand/longhand relationships are flexible enough that we could map `nowrap` in one to `no-wrap` in the other, but that seems very confusing to me.

**Comment by ExE-Boss** [comment]
@frivoal Well, if <https://github.com/w3c/csswg-drafts/issues/5382> is accepted, then it will make distinguishing the <code>[&lt;white‑space‑legacy&gt;]</code> `nowrap` value from the <code>[&lt;'text‑wrap'&gt;]</code> `no‑wrap` value much easier.

[&lt;white‑space‑legacy&gt;]: https://github.com/w3c/csswg-drafts/issues/5382#user-content-typedef-white-space-legacy
[&lt;'text‑wrap'&gt;]: https://github.com/w3c/csswg-drafts/issues/5382#user-content-propdef-text-wrap

**Comment by fantasai** [comment]
I'm very certain that we are not going to rename `nowrap` to `no-wrap`, simply for consistency with `white-space` and `flex-wrap`. At most we would add an alias--for all instances of `nowrap`.

**Comment by tabatkins** [comment]
Yes, we're currently consistently bad with `nowrap`, which sucks, but it's better than some properties having `nowrap` and others having `no-wrap`. Accepting both would be ideal.

**Comment by Crissov** [comment]
For a property called `text-wrap`, the frequently used keyword `none` would have felt more systematic to go with `wrap`, which could have been `auto` just as well. It only makes sense when considering the history and other properties.
